README.md

```diff
@@ -73,55 +85,145 @@ This will clone `10.15.6` to a new VM and, if there are differences from the bas
 
 > Check out the [examples directory](./examples) to see how port-forwarding and other options are used
 
-## Configuration
+## Builders 
+
+### veertu-anka-vm-create
+
+#### Required Configuration
+
+* `installer_app` (String)
+
+The path to a macOS installer. This process takes about 20 minutes. The resulting VM template name will be `{{vm_name}}-{{macOSVersion}}`. macOSVersion is pulled from the installer app.
+
+* `type` (String)
+
+Must be `veertu-anka-vm-create`.
+
+* `vm_name` (String)
+
+The name for the VM that is created. One is generated if not provided (`anka-packer-{10RandomCharacters}`).
 
-* `type` (required)
+#### Optional Configuration
 
-Must be `veertu-anka`.
+* `anka_password` (String)
 
-* `installer_app` (optional)
+Sets the password for the vm. Can also be set with `ANKA_DEFAULT_PASSWD` env var. Defaults to `admin`.
 
-The path to a macOS installer. This must be provided if `source_vm_name` isn't provided. This process takes about 20 minutes. The resulting VM template name will be `anka-packer-base-{macOSVersion}`.
+* `anka_user` (String)
 
-* `disk_size` (optional)
+Sets the username for the vm. Can also be set with `ANKA_DEFAULT_USER` env var. Defaults to `anka`.
+
+* `boot_delay` (String)
+
+The time to wait before running packer provisioner commands, defaults to `10s`.
+
+* `cpu_count` (String)
```

Comment:
I wonder if we should use this opportunity to switch to vcpu_count

Reply:
@NorseGaud is there a reason we're switching from cpu to vcpu? I'm not seeing that being used in other parts of anka?

Reply:
You're right about it not being used in the Anka CLI, but I'm pushing for changing it to vcpu in Anka 3.0 (still a work in progress) and elsewhere. VCPU is more accurate and eliminates any confusion and we'll want to change it eventually so might as well use this opportunity.

Reply:
@NorseGaud would this then be better suited in a PR that we can stage for when that time comes? I would think that could get confusing when people are trying to reference the documentation for what settings they need and incorrectly supply the wrong number of cores.

Reply:
post 2.0.0 release - I agree with your assessment there

Reply:
It's not super urgent for now. Asaf or I could make the change before 2.0.0 is fully released if you don't have the time. Just figured it'd be good to chat about. Major versions are great opportunities for refinement :)

Reply:
I think I misunderstood.. I can get the PR created post merge/release of 2.0.0 since I feel that it will line up more appropriately with the software changes coming with Anka 3.0 like you mentioned.

Reply:
No, I'd want this to happen pre-release of 2.0.0.

> My goal would be to not surprise users who start using 2.0.0 with a documentation/options change.